src/main/java/cubicchunks/CommonEventHandler.java

```diff
@@ -34,12 +37,19 @@
 
 	@SubscribeEvent
 	public void onWorldLoad(WorldEvent.Load evt) {
-		if (!(evt.getWorld().getWorldType() instanceof ICubicChunksWorldType) ||
-				evt.getWorld().provider.getDimension() != 0) {
+		if (!(evt.getWorld().getWorldType() instanceof ICubicWorldType)) {
 			return;
 		}
+
 		CubicChunks.LOGGER.info("Initializing world " + evt.getWorld() + " with type " + evt.getWorld().getWorldType());
 		ICubicWorld world = (ICubicWorld) evt.getWorld();
+
+		WorldType type = evt.getWorld().getWorldType();
```

Comment:
You can move this to the start of the method and use `type` instead of `evt.getWorld().getWorldType()` throughout the method.